### PR TITLE
AB#3130 - update home and mailing address with new figma

### PR DESCRIPTION
**Pull request by leiye29**
### Description

update home address page and mailing address page with https://www.figma.com/file/d8XksFd7eDtBF9V3IvaBeb/CDCP-Client-Portal-%28in-MSCA%29?type=design&node-id=3-4537&mode=design&t=eBkpwUJHsMVzR9U9-0

### Screenshots (if applicable)

image
image

### Checklist

- [x] I have tested the changes locally
- [ ] I have updated the documentation if necessary
- [ ] I have added/updated tests that prove my fix is effective or that my feature works
- [x] I have checked that my code follows the project's coding style by running `npm run format:check`
- [x] I have checked that my code contains no linting errors by running `npm run lint`
- [x] I have checked that my code contains no type errors by running `npm run typecheck`
- [x] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [x] I have checked that all e2e tests pass by running `npm run test:e2e`

### CODE REVIEW

File: frontend/public/locales/en/personal-information.json [hunk 1]
```diff
@@ -20,25 +20,28 @@
...
       },
       "field": {
         "address": "Address",
-        "city": "City",
-        "province": "Province/State",
-        "postal-code": "Postal code",
-        "country": "Country"
-      },
+        "address-note": "Include street number and name.",
+        "apartment": "Apartment, suite, etc. (opetional)",
```
Comment (gregory-j-baker) [inline frontend/public/locales/en/personal-information.json:31]:
```suggestion
        "apartment": "Apartment, suite, etc. (optional)",
```

File: frontend/public/locales/en/personal-information.json [hunk 2]
```diff
@@ -78,27 +81,28 @@
...
-      "copy-home-address-note": "We have the following home address on file, which will be used as your mailing address:",
       "field": {
         "address": "Address",
-        "city": "City",
-        "province": "Province/State",
-        "postal-code": "Postal code",
-        "country": "Country"
-      },
+        "address-note": "Include street number and name.",
+        "apartment": "Apartment, suite, etc. (opetional)",
```
Comment (gregory-j-baker) [inline frontend/public/locales/en/personal-information.json:92]:
```suggestion
        "apartment": "Apartment, suite, etc. (optional)",
```

**Review by gregory-j-baker — DISMISSED** [review]

File: frontend/public/locales/en/personal-information.json (hunk 2 above)
Comment (sebastien-comeau) [reply frontend/public/locales/en/personal-information.json:92]: Do we need `(optional)`?

File: frontend/app/routes/$lang+/_protected+/personal-information+/home-address+/edit.tsx [hunk 3]
```diff
@@ -244,15 +251,36 @@ export default function PersonalInformationHomeAddressEdit() {
...
+                label={t('personal-information:home-address.edit.field.postal-code')}
+                name="postalCode"
+                defaultValue={defaultValues.postalCode}
+                errorMessage={errorMessages.postalCode}
+                required={selectedCountry === CANADA_COUNTRY_ID || selectedCountry === USA_COUNTRY_ID}
+              />
+            </div>
           </div>
+
+          <InputCheckbox id="updateMailingAddress" name="updaetMailingAddress" value="copy" className="my-8">
```
Comment (nicholas-ly) [inline frontend/app/routes/$lang+/_protected+/personal-information+/home-address+/edit.tsx:271]:
```suggestion
          <InputCheckbox id="updateMailingAddress" name="updateMailingAddress" value="copy" className="my-8">
```

File: frontend/public/locales/en/personal-information.json (hunk 2 above)
Comment (leiye29) [reply frontend/public/locales/en/personal-information.json]: (optional) is in the Figma i am working on

Comment (leiye29) [reply frontend/public/locales/en/personal-information.json]: removed

**Review by nicholas-ly — APPROVED** [review]

**Pull request merged**